OpenLIFUPrePlanning/OpenLIFUPrePlanning.py

```diff
@@ -319,6 +323,15 @@ def onTargetListWidgetCurrentItemChanged(self, current:qt.QListWidgetItem, previ
         self.updateTargetPositionInputs()
         self.updateLockButtonIcon()
 
+    def onTargetListWidgetItemDataChanged(self, item:qt.QListWidgetItem):
+        node : vtkMRMLMarkupsFiducialNode = item.data(qt.Qt.UserRole)
+        node.SetName(item.text().replace(" ", "-")) # This becomes openlifu Point ID
+        node.SetNthControlPointLabel(0, item.text()) # This becomes openlifu Point name
+        node.InvokeEvent(SlicerOpenLIFUEvents.TARGET_NAME_MODIFIED_EVENT)
```

Comment:
Here is the _invocation_ of the custom event. In Qt terminology it's _emitting_ a signal